### PR TITLE
add note about micrometer

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

**Review by cartermp — APPROVED** [review]

**Review by trask — APPROVED** [review]

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 1]
```diff
@@ -23,3 +23,30 @@ OpenTelemetry.
...
+At a high level:
+
+OpenTelemetry Spring Boot starter
+
+- Natively uses the OpenTelemetry API, OpenTelemetry SDK and the OpenTelemetry
+  semantic conventions
+- Supported by the OpenTelemetry community (governed by the CNCF / Linux
+  Foundation)
+
+Micrometer [1], [2]
```
Comment (chalin) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:38]:
```suggestion
Micrometer [^1][^2]:
```

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 2]
```diff
@@ -23,3 +23,30 @@ OpenTelemetry.
...
+
+As the authors of the OpenTelemetry Spring Boot starter, we recommend using the
+OpenTelemetry Spring Boot starter. We have a great community behind it, and are
+continuously making improvements. If you try it out and have any problems or
+suggestions, please open an
+[issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues)
+in our repository.
+
+[1]: https://docs.micrometer.io/micrometer/reference/implementations/otlp.html
+[2]: https://docs.micrometer.io/tracing/reference/tracers.html
```
Comment (chalin) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:52]:
```suggestion
[^1]: https://docs.micrometer.io/micrometer/reference/implementations/otlp.html
[^2]: https://docs.micrometer.io/tracing/reference/tracers.html
```

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 3]
```diff
@@ -23,3 +23,30 @@ OpenTelemetry.
    - **Spring Boot configuration files** (`application.properties`,
      `application.yml`) to configure the OpenTelemetry Spring Boot starter which
      doesn't work with the OpenTelemetry Java agent
+
+## Comparison to Micrometer
+
+At a high level:
+
+OpenTelemetry Spring Boot starter
```
Comment (chalin) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:31]:
```suggestion
OpenTelemetry Spring Boot starter:
```

**Review by chalin — CHANGES_REQUESTED** [review]
As mention on Slack. I think that the entire content of this section should be moved one level up, into the Java landing page (or it's own Comparing page).

**Comment by trask** [comment]
> As mention on Slack. I think that the entire content of this section should be moved one level up, into the Java landing page (or it's own Comparing page).

I think I prefer its own Comparing page under Spring Boot Starter

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ aliases:
   - /docs/zero-code/java/spring-boot
 ---
 
+## Java agent vs Spring Boot starter
```
Comment (theletterf) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:9]:
```suggestion
## Java agent versus Spring Boot starter
```

https://developers.google.com/style/word-list#vs

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 5]
```diff
@@ -23,3 +25,32 @@ OpenTelemetry.
...
      doesn't work with the OpenTelemetry Java agent
+
+## Comparison to Micrometer
+
+At a high level, this section compares OpenTelemetry Spring Boot starter and
+Micrometer.
+
+OpenTelemetry Spring Boot starter:
+
+- Natively uses the OpenTelemetry API, OpenTelemetry SDK and the OpenTelemetry
```
Comment (theletterf) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:36]:
```suggestion
- Natively uses the OpenTelemetry API, OpenTelemetry SDK, and the OpenTelemetry
```

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 6]
```diff
@@ -23,3 +25,32 @@ OpenTelemetry.
...
+  semantic conventions
```
Comment (theletterf) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:37]:
```suggestion
  semantic conventions.
```

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 7]
```diff
@@ -23,3 +25,32 @@ OpenTelemetry.
...
+- Supported by the OpenTelemetry community (governed by the CNCF / Linux
+  Foundation)
```
Comment (theletterf) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:39]:
```suggestion
  Foundation).
```

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 8]
```diff
@@ -23,3 +25,32 @@ OpenTelemetry.
...
+
+Micrometer [^1][^2]:
+
+- An independent standalone observability system
+- Offers optional bridges into parts of the OpenTelemetry ecosystem
+- Supported by the Micrometer community (governed by Broadcom Inc)
```
Comment (theletterf) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:45]:
```suggestion
- An independent standalone observability system.
- Offers optional bridges into parts of the OpenTelemetry ecosystem.
- Supported by the Micrometer community (governed by Broadcom Inc).
```

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 9]
```diff
@@ -23,3 +25,32 @@ OpenTelemetry.
...
+
+As the authors of the OpenTelemetry Spring Boot starter, we recommend using the
+OpenTelemetry Spring Boot starter. We have a great community behind it, and are
+continuously making improvements. If you try it out and have any problems or
+suggestions, please open an
+[issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues)
+in our repository.
```
Comment (theletterf) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:52]:
```suggestion
As the authors of the OpenTelemetry Spring Boot starter, we recommend using the
OpenTelemetry Spring Boot starter. We have a great community behind it, and are
continuously making improvements. If you try it out and have any problems or
suggestions, open an
[issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues)
in our repository.
```

**Review by theletterf — CHANGES_REQUESTED** [review]
There's something in adding a recommendation in user docs that is a bit jarring to me (using Spring Boot starter instead of another solution). I think docs should be more neutral. Perhaps the recommendation or even the whole content could go into a blog post. @svrnm What do you reckon?

**Comment by zeitlinger** [comment]
> There's something in adding a recommendation in user docs that is a bit jarring to me (using Spring Boot starter instead of another solution). I think docs should be more neutral. Perhaps the recommendation or even the whole content could go into a blog post. @svrnm What do you reckon?

the tone should be neutral
we keep getting those questions - so it would be helpful to have an easy-to-find answer

**Comment by chalin** [comment]
Here's a proposal: @zeitlinger make the minor suggested edits, we land this PR, and after that look at moving the content to a dedicated comparing page. WDYT?

**Comment by chalin** [comment]
100% agree with @theletterf comment about neutrality -- and thanks for bring that up, it had bothered me too.

**Comment by svrnm** [comment]
on the topic of neutrality: since this seems to be a frequently asked question it can/should live in the docs not in the blog, I actually think we can have more of that in the future, but key is that the wording is friendly and embracing the other open source solution as well. I will reread the PR under that aspect once again

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 10]
```diff
@@ -23,3 +25,34 @@ OpenTelemetry.
    - **Spring Boot configuration files** (`application.properties`,
      `application.yml`) to configure the OpenTelemetry Spring Boot starter which
      doesn't work with the OpenTelemetry Java agent
+
+## Comparison to Micrometer
+
+At a high level, this section compares OpenTelemetry Spring Boot starter and
+Micrometer.
```
Comment (svrnm) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:32]:
```suggestion
A question asked frequently is how [Micrometer](https://micrometer.io/) and the OpenTelemetry Spring Boot Starter differentiate. At a high level, the differences are as follows:
```

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 11]
```diff
@@ -23,3 +25,34 @@ OpenTelemetry.
...
+- An independent standalone observability system.
+- Offers optional bridges into parts of the OpenTelemetry ecosystem.
+- Supported by the Micrometer community (governed by Broadcom Inc).
+
+As the authors of the OpenTelemetry Spring Boot starter, we recommend using the
+OpenTelemetry Spring Boot starter. We have a great community behind it, and are
+continuously making improvements. If you try it out and have any problems or
+suggestions, open an
+[issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues)
+in our repository.
```
Comment (svrnm) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:52]:
```suggestion
Both solutions are open source and support different exporters, including OTLP and Prometheus,
and both are compatible for tracing use cases. This means, that they fit equally into an existing 
observability architecture. It is up to the end user to decide, which solution satisfies their requirements better. 
```

This is the sentence I would remove in the light of "neutrality". It's very obvious that we recommend the otel solution, so we don't have to say that explicitly.

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md [hunk 12]
```diff
@@ -23,3 +25,34 @@ OpenTelemetry.
...
+
+- Natively uses the OpenTelemetry API, OpenTelemetry SDK, and the OpenTelemetry
+  semantic conventions.
+- Supported by the OpenTelemetry community (governed by the CNCF / Linux
+  Foundation).
+
+Micrometer [^1][^2]:
+
+- An independent standalone observability system.
+- Offers optional bridges into parts of the OpenTelemetry ecosystem.
```
Comment (svrnm) [inline content/en/docs/zero-code/java/spring-boot-starter/_index.md:44]:
```suggestion
- Offers optional bridges into OpenTelemetry Tracing and Metrics.
```

**Review by svrnm — CHANGES_REQUESTED** [review]
Taking a closer look under the "neutrality" perspective, see my suggestions.

A general comment: the differences are very high level, so as an end user I might want to know even more. Can we expand on that a little bit (e.g. my understanding is that micrometer is a facade around different metric/tracing solutions ("Micrometer provides a simple facade over the instrumentation clients for the most popular observability systems"), I think that's a key differentiator, if someone thinks they do not want to be "vendor-locked" by opentelemetry, they can have another layer of abstraction.

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md (hunk 12 above)
Comment (zeitlinger) [reply content/en/docs/zero-code/java/spring-boot-starter/_index.md:43]: you also don't get SDK features

File: content/en/docs/zero-code/java/spring-boot-starter/_index.md (hunk 11 above)
Comment (zeitlinger) [reply content/en/docs/zero-code/java/spring-boot-starter/_index.md:51]: It sounds a bit like we would not be convinced that our solution is the better one.

We do try to become the standard API in the java world - can we say that in a friendly way?

**Comment by zeitlinger** [comment]
> if someone thinks they do not want to be "vendor-locked" by opentelemetry, they can have another layer of abstraction.

the message of this PR is that you're not locked, because only otel is governed by an independent body

**Comment by svrnm** [comment]
@zeitlinger can we make any progress on this? If I remember correctly @trask wanted to take another look at this?

**Comment by zeitlinger** [comment]
yes, @trask is still on it

**Comment by cartermp** [comment]
Closing out old PR -- I realize we wanted to say something here but it's been a few months, so feel free to re-open if you think it's still worth it

**Pull request closed without merging**